### DISCUSSION TITLE
2024.03.12(화)

### DISCUSSION THREAD

**Discussion by Crack-Egg** [Party-Time-Job/Party-Time-Job#34] [회의록]
## 1. 디자인

앱 느낌으로 디자인 변경 예정 → 모든 기능이 완성되고 나서 디자인 변경하기
[https://www.behance.net/gallery/145396975/PLANTECO](url)

로고 아이콘 → 협상 + 파티 느낌을 주는 아이콘
색상과 명암은 추후 변경

## 2. 폴더 구조

Feature-Sliced Design 구조
폴더 구조 만들고 PR 올릴 예정. 그 전까지 되도록 PR 올리지 않기!

**Comment by dongqui** [comment]
지식왕
